convert2rhel/pkghandler.py

```diff
@@ -366,6 +367,14 @@ def get_packager(pkg_obj):
     return packager.split("<", 1)[0].rstrip()
 
 
+def get_vendor(pkg_obj):
+    # The vendor may not be set for all packages
+    if hasattr(pkg_obj, "vendor") and pkg_obj.vendor:
+        return pkg_obj.vendor
+    else:
+        return "N/A"
```

Comment:
Nit:
I think more pythonic will be:
```python
try:
    return pkg_obj.vendor
except AttributeError:
    return "N/A"
```

Reply:
Nice catch!

Reply:
Actually, I wrote like that, because, for some package vendor can come as None, use try..block will return None, on my code, I check it..